Return message if a pkg can't be found

csgillespie opened the pull request:
Currently if a package can't be found, the message is misleading, e.g.

```
R> library("ggplot3")
Error in library("ggplot3") : there is no package called 'ggplot3'
Can now rerun
library("ggplot3")
```
This PR gives a more better message

```
R> library("ggplot3")
Error in library("ggplot3") : there is no package called 'ggplot3'
Unable to install ggplot3
````

csgillespie committed:
Return message if the pkg can't be found

jimhester commented:
:+1:, but maybe the message should be something like

> Package 'ggplot3' does not exist

csgillespie committed:
Return early
Fix #3

csgillespie commented:
The commit looks more complicated than it actually is. 

1. Changed
```
 if (length(pkg)  > 0) {

}
```
to  `if (length(pkg) == 0L) return()`. This is slightly more efficient, but also removes an indent.

2. Update message to         `message("Package '", pkg, "' does not exist")`

3. Fixed #3 via
```
    if (i != "N") {
      devtools::install_github(gh_pkgs$pkg_location[matches[as.integer(i)]])
    } else {
      return()
    }
```
I don't think a return message is need here.

jimhester commented:
Thanks!
